forest-node/ansible/roles/docker/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Whew! What this means is I should use terraform to initialize a `server using Docker image` and then run forest on it. Not a bad idea but I do not think the way we are running it now is a problem because this is a straight lift from docker docs. 

However, I'd try this idea, I just have to be sure it can offer enough server, CPU and RAM expectations